person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] To implement the `has-many` side of the relationship between `Person` and `Rentals`, method for adding rentals should be present in this `Person` class. Something like so;
```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```